backend/cmd/api/main.go

```diff
@@ -172,7 +173,20 @@ func main() {
 	queries := sqlc.New(db)
 	queries.SetImageCDNDomain(config.CDNConfig.ImageCDNDomain)
 	authClient := auth0.New(config.Auth0)
-	membersClient := members.New(config.Members, authClient)
+
+	cb := gobreaker.NewCircuitBreaker(gobreaker.Settings{
+		Name:    "Members",
+		Timeout: time.Second * 2,
+		ReadyToTrip: func(counts gobreaker.Counts) bool {
+			failureRatio := float64(counts.TotalFailures) / float64(counts.Requests)
```

Comment:
Is counts.Requests guaranteed to to be `!= 0`?

Reply:
Think I'm going to keep it as simple as possible for the initial implementation.